src/MigrationsOrganiserServiceProvider.php

```diff
@@ -28,7 +28,7 @@ protected function registerMigrator()
         $this->app->singleton('migrator', function ($app) {
             $repository = $app['migration.repository'];
 
-            return new Migrator($repository, $app['db'], $app['files']);
+            return new Migrator($repository, $app['db'], $app['files'], $app['events']);
```

Comment:
Passing same arguments as in the original definition https://github.com/laravel/framework/blob/c0dc258eef3ec1593b43a0205aa4da003d424fc3/src/Illuminate/Database/MigrationServiceProvider.php#L81